packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -51,9 +51,7 @@ export const PolygonGraph = (props: Props) => {
                 points={[...points]}
                 color="var(--movable-line-stroke-color)"
                 svgPolygonProps={{
-                    strokeWidth: active
-                        ? "var(--movable-line-stroke-weight-active)"
-                        : "var(--movable-line-stroke-weight)",
+                    style: {fill: "transparent"},
```

Comment:
Setting the color on line 52 starts the graph out with a colored background. Removing this line turns the graph gray though. If we leave line 52 and just change the fill here, it works as needed.